src/Fixer/Whitespace/IndentationTypeFixer.php

```diff
@@ -70,7 +72,9 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 }
 
                 // change indent to expected one
-                $content = preg_replace('/^    /m', $this->whitespacesConfig->getIndent(), $content);
+                $content = preg_replace_callback('/^(?:    )+/m', function ($matches) use ($expectedIndent) {
```

Comment:
Does this play nicely with the double space indentation option?
Like doesn't this fix 2x4 spaces to 2x2 spaces and on second run the 2x2 spaces to 2 spaces?

Reply:
no clue, I didn't touch 2x spaces here, not sure do we have tests for those.
maybe let us drop 2x spaces support in general? I remember I added tests with tabs for all whitespace-touching fixers and done integration tests for them, but probably noone has run it for 2 spaces

Reply:
I recalled this https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2650 which seems related to this issue as well.

Reply:
let us not fix everything at once. until we will have at least integration test of 2 spaces, I can't say anything about quality of supporting it.

Reply:
all right, I'll only check if you fix causes regression with two-space config than, :+1:

Reply:
green ?

Reply:
I didn't find any new issue with the double spaces, so :+1:  from me